src/utils/signTypedData.ts

```diff
@@ -50,6 +51,26 @@ export async function signTypedData(
       return { cancelled: true };
     }
 
+    if(!signer)
+      throw new Error(typeof error === 'string' ? error : 'An error occured.');
```

Comment:
can we make the error here more specific since we know the reason, something like "Error: signer not defined"?

Reply:
I didn't want to change the error you guys had already put there (when all `try`'s fail) but we could put something more precise

Reply:
I see, makes sense :+1: